tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Shared/SpanPointers.cs

```diff
@@ -54,6 +57,41 @@ public static void AddS3SpanPointer(Span span, string bucketName, string key, st
         span.AddLink(spanLink);
     }
 
+    // DynamoDB hashing rules: https://github.com/DataDog/dd-span-pointer-rules/blob/main/AWS/DynamoDB/Item/README.md
+    public static void AddDynamoDbSpanPointer(Span span, string tableName, IDynamoDbKeysObject keys)
+    {
+        if (!keys.KeyNames.Any())
```

Comment:
Unsure really, but maybe KeyNames should be checked for `null` in case the Duck gives us a null type (I can't find the underlying class for it)